core/domain/story_jobs_one_off_test.py

```diff
@@ -251,3 +251,45 @@ def _mock_get_story_by_id(unused_story_id):
         for x in output:
             self.assertRegexpMatches(
                 x, 'Expected description to be a string, received 123')
+
+
+class PopulateStoryThumbnailSizeOneOffJobTests(test_utils.GenericTestBase):
+
+    ALBERT_EMAIL = 'albert@example.com'
+    ALBERT_NAME = 'albert'
+
+    STORY_ID = 'story_id'
+
+    def setUp(self):
+        super(PopulateStoryThumbnailSizeOneOffJobTests, self).setUp()
+
+        self.signup(self.ALBERT_EMAIL, self.ALBERT_NAME)
+        self.albert_id = self.get_user_id_from_email(self.ALBERT_EMAIL)
+        self.TOPIC_ID = topic_fetchers.get_new_topic_id()
+        self.story_id_1 = 'story_id_1'
+        self.save_new_topic(
+            self.TOPIC_ID, self.albert_id, name='Name',
+            description='Description',
+            canonical_story_ids=[self.story_id_1]
+        )
+        self.process_and_flush_pending_mapreduce_tasks()
+
+    def test_thumbnail_size_job_thumbnail_size_is_present(self):
```

Comment:
We have to run the job in each of the tests -- like [here](https://github.com/oppia/oppia/blob/7dac6b7c187c5944da9ec93030f098f99f1c150e/core/domain/story_jobs_one_off_test.py#L104), and then make sure the assertions are right by comparing (key, values) from the job output.

Reply:
updated.